src/containers/Tenant/Query/i18n/en.json

```diff
@@ -54,8 +54,8 @@
 
   "filter.text.placeholder": "Search by query text...",
 
-  "gear.tooltip": "Query execution settings have been changed for ",
-  "banner.query-settings.message": "Query results are displayed for ",
+  "gear.tooltip": "Query execution settings: ",
```

Comment:
In gear tooltip we show only changed settings, so previous text was more applicable